web/portal/client/cypress/e2e/Friend/CallForwardSettings/CallForwardSettings.cy.js

```diff
@@ -0,0 +1,73 @@
+import CallForwardSettingCollection from '../../../fixtures/CallForwardSetting/getCollection.json';
+import NewCallForwardSetting from '../../../fixtures/CallForwardSetting/post.json';
+import {
+  deleteCallForwardSetting,
+  putCallForwardSetting,
+} from '../../CallForwardSetting/CallForwardSetting.tests';
+
+describe('CallForwardSetting', () => {
+  beforeEach(() => {
+    cy.prepareGenericPactInterceptors('Friend-CallForwardSetting');
+    cy.before();
+
+    cy.contains('Routing endpoints').click();
+    cy.contains('Friends').click();
+
+    cy.get('header').should('contain', 'Friends');
+
+    cy.get('svg[data-testid="PhoneForwardedIcon"]').first().click();
+
+    cy.get('table').should(
+      'contain',
+      CallForwardSettingCollection.body[0].numberValue
+    );
+  });
+
+  ///////////////////////
+  // POST
+  ///////////////////////
+  it('add CallForwardSetting', () => {
+    cy.usePactIntercept(
+      {
+        method: 'POST',
+        url: '**/api/client/call_forward_settings*',
+        response: NewCallForwardSetting.response,
+        matchingRules: NewCallForwardSetting.matchingRules,
+      },
+      'createCallForwardSetting'
+    );
+
+    cy.get('[aria-label=Add]').click();
+
+    const {
+      callTypeFilter,
+      callForwardType,
+      targetType,
+      numberValue,
+      friend,
+      extension,
+      voicemail,
+      enabled,
+    } = NewCallForwardSetting.request;
+    cy.fillTheForm({
+      callTypeFilter,
+      callForwardType,
+      targetType,
+      numberValue,
+      friend,
+      extension,
+      voicemail,
+      enabled,
+    });
```

Comment:
In a separate function please, or if you prefer, set the other logic in this file, but do not create this kind of mixed logic